pkg/k8s/client.go

```diff
@@ -0,0 +1,92 @@
+package k8s
+
+import (
+	"context"
+
+	appsv1 "k8s.io/api/apps/v1"
+	apiv1 "k8s.io/api/core/v1"
+	apiextensionsv1 "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	k8sclientset "k8s.io/apiextensions-apiserver/pkg/client/clientset/clientset"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	k8stypes "k8s.io/apimachinery/pkg/types"
+	"k8s.io/utils/pointer"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+// Perform a compile time check.
+var _ Client = &KubeClient{}
+
+const DestinationRuleCrdName string = "destinationrules.networking.istio.io"
```

Comment:
Can you please add a comment above this const explaining what does it mean and why is it needed?

Reply:
Its added in next PR [here](https://github.com/kyma-project/nats-manager/blob/5969f82f8a2347d8c0798a16a12bfcf1488a4f73/pkg/k8s/types.go).